src/GenAPI/Microsoft.DotNet.GenAPI/SyntaxGeneratorExtensions.cs

```diff
@@ -66,6 +66,19 @@ public static SyntaxNode DeclarationExt(this SyntaxGenerator syntaxGenerator, IS
                 }
             }
 
+            if (symbol.Kind == SymbolKind.Event)
```

Comment:
You can probably replace this `if` with `if (symbol is IEventSymbol eventSymbol)`, and avoid the nested conditional

Reply:
Updated:
```C#
if (symbol is IEventSymbol eventSymbol && !eventSymbol.IsAbstract) {...}
```